src/components/CardPreview.vue

```diff
@@ -5,7 +5,7 @@
       <button id="delete" @click="deleteCard">&times;</button>
     </div>
     <Backdrop v-if="show" @clicked="toggleCard"/>
-    <Card v-if="show" v-bind:card="card" v-on:close="toggleCard"/>
+    <Card v-if="show" v-bind:card="card" @close="toggleCard" @saved="$emit('saved')"/>
```

Comment:
als je 2 keer een $emit doet, kun je het dan niet beter via de store oid doen?

Reply:
Hmm. Tja ik geef toe dat dit niet zo netjes er uit ziet. Maar het liefste houd ik wel zoveel mogelijk uit de store.
Denk ook dat het wel wat moeilijker zou zijn om dit via de store te doen. Omdat je een methode in een andere component aan het aanspreken bent enzo.

Reply:
fair